### ISSUE TITLE
Add link to elequent javascript to readme JS section?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/start-here#10]
While the @iteles notes on "The Good Parts" are great! Some people have found elequent javascript quite useful: http://eloquentjavascript.net/ should we add it?

**Comment by iteles** [comment]
Absolutely. We've met lots of people through [Founders & Coders](www.foundersandcoders.org) who feel The Good Parts is too dense and [Eloquent JS](http://eloquentjavascript.net/) is much easier to follow.

Also, my Good Parts notes really need tidying up urgently so it would be good to have another resource. I should work on that this weekend for a few hours, they're not entirely presentable!
